frontend_tests/node_tests/people.js

```diff
@@ -533,8 +572,8 @@ run_test('message_methods', () => {
         ],
         avatar_url: 'legacy.png',
     };
-    assert.equal(people.pm_with_url(message), '#narrow/pm-with/452-athens');
-    assert.equal(people.pm_perma_link(message), '#narrow/pm-with/30,452-pm');
+    assert.equal(people.pm_with_url(message), '#narrow/pm-with/302-athens');
```

Comment:
There are some hardcoded ids still left over, which we can remove later.

Reply:
Ah, some of those are semi-intentional, as I prefer the tedium of fixing them (like I just did) to the risk of false positives (where you're computing expected values).